cloud/pkg/router/provider/eventbus/eventbus.go

```diff
@@ -60,20 +61,20 @@ func (factory *eventbusFactory) GetSource(ep *v1.RuleEndpoint, sourceResource ma
 }
 
 func (eb *EventBus) RegisterListener(handle listener.Handle) error {
-	listener.MessageHandlerInstance.AddListener(fmt.Sprintf("%s/node/%s/%s/%s", "bus", eb.nodeName, eb.namespace, eb.subTopic), handle)
+	listener.MessageHandlerInstance.AddListener(path.Join("bus/node", eb.nodeName, eb.namespace, eb.subTopic), handle)
```

Comment:
can you pls share some data on how the performance got improved?

Reply:
```golang
var s1t time.Time
var used time.Duration

s1t = time.Now()
for i := 0; i < 100000; i++ {
    fmt.Sprintf("node/%s/%s/%s", "nodeName", "namespace", "subTopic")
}
used = time.Since(s1t)
fmt.Println("time cost of `sprintf`:", used)

s1t = time.Now()
for i := 0; i < 100000; i++ {
    path.Join("node", "nodeName", "namespace", "subTopic")
}
used = time.Since(s1t)
fmt.Println("time cost of `join`:", used)
```

Run 3 times pirnt out: 

```
=== RUN   TestPathJoin
time cost of `sprintf`: 15.840042ms
time cost of `join`: 12.771455ms
--- PASS: TestPathJoin (0.03s)

=== RUN   TestPathJoin
time cost of `sprintf`: 14.852592ms
time cost of `join`: 12.130716ms
--- PASS: TestPathJoin (0.03s)

=== RUN   TestPathJoin
time cost of `sprintf`: 19.307545ms
time cost of `join`: 14.843423ms
--- PASS: TestPathJoin (0.03s)
```